fastchat/serve/inference.py

```diff
@@ -67,6 +67,9 @@ def load_model(model_path, device, num_gpus, max_gpu_memory="13GiB",
         replace_llama_attn_with_non_inplace_operations()
     else:
         raise ValueError(f"Invalid device: {device}")
+    
+    if load_8bit:
+        return load_compress_model(model_path=model_path,device=device)
```

Comment:
1. Raise warnings if it is not compatible with multiple GPUs and users use multiple GPUs.
2. (Optional) Fallback to the old implementation if people use multiple GPUs.